src/server/scripts/EasternKingdoms/BlackrockMountain/BlackrockDepths/blackrock_depths.cpp

```diff
@@ -442,622 +442,17 @@ enum DughalQuests
-            switch (waypointId)
-            {
-                case 0:
-                    me->Say(SAY_DUGHAL_FREE, LANG_UNIVERSAL, PlayerGUID);
-                    break;
-                case 1:
-                    instance->SetData(DATA_DUGHAL, ENCOUNTER_STATE_OBJECTIVE_COMPLETED);
-                    break;
-                case 2:
-                    me->SetVisible(false);
-                    me->SetFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NOT_SELECTABLE);
-                    me->SetFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NON_ATTACKABLE);
-                    instance->SetData(DATA_DUGHAL, ENCOUNTER_STATE_ENDED);
-                    break;
-            }
-        }
-
-        void EnterCombat(Unit* who) { }
-        void Reset() { }
-
-        void JustDied(Unit* killer)
-        {
-            if (IsBeingEscorted && killer == me)
-            {
-                me->SetVisible(false);
-                me->SetFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NOT_SELECTABLE);
-                me->SetFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NON_ATTACKABLE);
-                instance->SetData(DATA_DUGHAL, ENCOUNTER_STATE_ENDED);
-            }
-        }
-
-        void UpdateAI(uint32 diff)
-        {
-            if (instance->GetData(DATA_QUEST_JAIL_BREAK) == ENCOUNTER_STATE_NOT_STARTED) return;
-            if ((instance->GetData(DATA_QUEST_JAIL_BREAK) == ENCOUNTER_STATE_IN_PROGRESS || instance->GetData(DATA_QUEST_JAIL_BREAK) == ENCOUNTER_STATE_FAILED || instance->GetData(DATA_QUEST_JAIL_BREAK) == ENCOUNTER_STATE_ENDED)&& instance->GetData(DATA_DUGHAL) == ENCOUNTER_STATE_ENDED)
-            {
-                me->SetVisible(false);
-                me->SetFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NOT_SELECTABLE);
-                me->SetFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NON_ATTACKABLE);
-            }
-            else
-            {
-                me->SetVisible(true);
-                me->RemoveFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NOT_SELECTABLE);
-                me->RemoveFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NON_ATTACKABLE);
-            }
-            npc_escortAI::UpdateAI(diff);
-        }
-    };
-};
-
- */
-
-// npc_marshal_windsor
-#define SAY_WINDSOR_AGGRO1          "You locked up the wrong Marshal. Prepare to be destroyed!"
-#define SAY_WINDSOR_AGGRO2          "I bet you're sorry now, aren't you !?!!"
-#define SAY_WINDSOR_AGGRO3          "You better hold me back $N or they are going to feel some prison house beatings."
-#define SAY_WINDSOR_1               "Let's get a move on. My gear should be in the storage area up this way..."
-#define SAY_WINDSOR_4_1             "Check that cell, $N. If someone is alive in there, we need to get them out."
-#define SAY_WINDSOR_4_2             "Get him out of there!"
-#define SAY_WINDSOR_4_3             "Good work! We're almost there, $N. This way."
-#define SAY_WINDSOR_6               "This is it, $N. My stuff should be in that room. Cover me, I'm going in!"
-#define SAY_WINDSOR_9               "Ah, there it is!"
-
 enum MarshalWindsor
```

Comment:
Maybe need to delete? and same enums

Reply:
Okay, I'll check. Maybe they need to be eliminated.

Reply:
Confirmed, there were 3 enum that were not used. Thanks for the observation.